packages/editor/src/components/collab-sidebar/comment-author-info.js

```diff
@@ -69,6 +70,8 @@ function CommentAuthorInfo( { avatar, name, date } ) {
 						dateTimeFormat,
 						date ? getDate( date ) : currentDate
 					) }
+					<br />
+					{ humanTimeDiff( date ? getDate( date ) : currentDate ) }
```

Comment:
```suggestion
					{ date && (
						<>
							<br />
							{ humanTimeDiff( getDate( date ) ) }
						</>
					) }
```

Sorry, this code seems to produce a weird result on the comment form:

<img width="300" height="246" alt="Image" src="https://github.com/user-attachments/assets/ddd6e108-bb87-43d0-a2ef-87716a9bf419" />

I think the "a second ago" is necessary.

Reply:
hi @t-hamano 
After adding this code no any time like "a second ago" visible on latest open comment form

```
{ date && (
						<>
							<br />
							{ humanTimeDiff( getDate( date ) ) }
						</>
					) }
```
<img width="268" height="558" alt="Screenshot 2025-09-17 at 8 03 24 PM" src="https://github.com/user-attachments/assets/ab001eb1-2936-4de9-b53b-5dab20bf3ed6" />